### PR TITLE
[SASS-5591: Create "warning page - Off-payroll working status?" page - OPW]

### PR DESCRIPTION
**Pull request by shehzad-ismail8**
### Description
Created a new page as part of the OPW journey as described in the Jira ticket

Add a link to the relevant story in Jira
[SASS-5591](https://jira.tools.tax.service.gov.uk/browse/SASS-5591)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [ ]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

### CODE REVIEW

**Review by kannanramakrishnan — CHANGES_REQUESTED** [review]
please check and update review comments

File: app/controllers/offPayrollWorking/ChangeYourEmploymentDetailsController.scala [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import scala.concurrent.{ExecutionContext, Future}
+
+class ChangeYourEmploymentDetailsController @Inject()(mcc: MessagesControllerComponents,
+                                                      authAction: AuthorisedAction,
+                                                      view: ChangeYourEmploymentDetailsView)
+                                                     (implicit appConfig: AppConfig, ec: ExecutionContext) extends FrontendController(mcc)
+  with I18nSupport with SessionHelper {
+
+  def show(taxYear: Int): Action[AnyContent] = (authAction andThen TaxYearAction.taxYearAction(taxYear)).async { implicit request =>
+    val headingIndividual = "employment.employerOpw.warning.heading.individual"
```
[the remark this answers is not in the document]
Comment (shehzad-ismail8) [reply app/controllers/offPayrollWorking/ChangeYourEmploymentDetailsController.scala]: Amended

File: conf/messages [hunk 2]
```diff
@@ -192,6 +192,16 @@ employment.employerOpw.heading.agent = Does your client agree with ABC Digital
...
 employment.employerOpw.error.agent = Select yes if your client agrees with ABC Digital’s decision
 
+employment.employerOpw.warning.title.individual = Do you want to change your employment details?
+employment.employerOpw.warning.heading.individual = Do you want to change your employment details?
+employment.employerOpw.warning.title.agent = Do you want to change your client’s employment details?
+employment.employerOpw.warning.heading.agent = Do you want to change your client’s employment details?
+employment.employerOpw.warning.p1 = You are about to change information ABC Digital sent to HMRC:
+employment.employerOpw.warning.b1 = Off-payroll working status
+employment.employerOpw.warning.p2 = HMRC may review this change.
+employment.employerOpw.warning.button = Confirm
```
[the remark this answers is not in the document]
Comment (shehzad-ismail8) [reply conf/messages]: Amended

File: conf/messages.cy [hunk 3]
```diff
@@ -192,6 +192,16 @@ employment.employerOpw.heading.agent = A yw’ch cleient yn cytuno â phenderfyn
...
 employment.employerOpw.error.agent = Dewiswch ‘Iawn’ os yw’ch cleient yn cytuno â phenderfyniad ABC Digital Ltd
 
+employment.employerOpw.warning.title.individual = A ydych am newid eich manylion cyflogaeth?
+employment.employerOpw.warning.heading.individual = A ydych am newid eich manylion cyflogaeth?
+employment.employerOpw.warning.title.agent = A ydych am newid manylion cyflogaeth eich cleient?
+employment.employerOpw.warning.heading.agent = A ydych am newid manylion cyflogaeth eich cleient?
+employment.employerOpw.warning.p1 = Rydych ar fin newid manylion a anfonwyd at CThEF gan ABC Digital Ltd:
+employment.employerOpw.warning.b1 = Statws gweithio oddi ar y gyflogres
+employment.employerOpw.warning.p2 = Mae’n bosibl y bydd CThEF yn adolygu’r newid hwn
+employment.employerOpw.warning.button = Cadarnhau
```
[the remark this answers is not in the document]
Comment (shehzad-ismail8) [reply conf/messages.cy]: Amended

**Review by kannanramakrishnan — APPROVED** [review]

**Pull request closed without merging**